client/webserver/site/src/font/icomoon.svg

```diff
@@ -7,14 +7,15 @@
+<glyph unicode="&#xea0a;" glyph-name="plus" d="M992.97 460.637h-352.344v352.344c0 17.689-14.342 32.031-32.031 32.031h-192.188c-17.689 0-32.031-14.342-32.031-32.031v-352.344h-352.344c-17.689 0-32.031-14.342-32.031-32.031v-192.188c0-17.689 14.342-32.031 32.031-32.031h352.344v-352.344c0-17.689 14.342-32.031 32.031-32.031h192.188c17.689 0 32.031 14.342 32.031 32.031v352.344h352.344c17.689 0 32.031 14.342 32.031 32.031v192.188c0 17.689-14.342 32.031-32.031 32.031z" />
+<glyph unicode="&#xea0b;" glyph-name="minus" d="M0 428.606v-192.188c0-17.689 14.342-32.031 32.031-32.031h960.938c17.689 0 32.031 14.342 32.031 32.031v192.188c0 17.689-14.342 32.031-32.031 32.031h-960.938c-17.689 0-32.031-14.342-32.031-32.031z" />
+<glyph unicode="&#xea10;" glyph-name="checkmark" d="M864.845 716.888l-480.469-480.469-224.219 224.219-160.156-160.156 384.375-384.375 640.626 640.626z" />
```

Comment:
Instead of replacing the entire file, copy just the new glyph element into the old file. This retains order, helps with review, and prevents drifting of attributes.